vmdb/app/helpers/provider_foreman_helper/textual_summary.rb

```diff
@@ -0,0 +1,37 @@
+module ProviderForemanHelper
+  module TextualSummary
```

Comment:
The summary helper methods can get quite lengthy,  so they were split out
in most every controller.  This is consistent with the other helpers,  so
should we break consistency? Granted they still need refactoring again, and
probably should become presentors or some other kind of object, but keeping
them separate might make that easier.
 On Mar 17, 2015 10:52 AM, "Aaron Patterson" <notifications@github.com>
wrote:

> In vmdb/app/helpers/provider_foreman_helper/textual_summary.rb
> <https://github.com/ManageIQ/manageiq/pull/1981#discussion_r26579116>:
>
> > @@ -0,0 +1,37 @@
> > +module ProviderForemanHelper
> > +  module TextualSummary
>
> If we're only using this module in one place, then we should put the
> methods inside ProviderForemanHelper
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/1981/files#r26579116>.
>